### DISCUSSION TITLE
01-03 강의 준비에 질문이 있습니다.

### DISCUSSION THREAD

**Discussion by hyunkyung13306** [tedilabs/fastcampus-devops#41] [Q&A] [answered]
AWS에서 pem 키를 다운받아 chmod로 ubuntu에서 
강사님과 똑같이 chmod 600 fastcampus.pem 키를 입력했는데 먹질 않아요
window11인데 권한 설정을 어떻게 해야 하는지 모르겠습니다.
fastcampus.pem* << 파일 명 뒤에 * 표시가 되어있고 권한 변경이 안됩니다.
어떻게 해야 하나요?

**Comment by posquit0** [comment, marked as answer]
안녕하세요,
이슈 확인을 위해 질문을 드리고 싶습니다.

- Windows WSL을 통해서 우분투 배시 셸에서 실행한 명령일까요?
- `pwd` 명령어를 통해 현재 경로 확인을 하고 싶습니다.
- 우분투의 파일 시스템 경로 상에서 명령어를 수행했는지 윈도우의 c 드라이브 경로에서 명령어를 수행했는지 확인이 필요합니다. 권한 관련 명령은 우분투 파일시스템 경로 상에서 실행되어야 합니다!

**Reply by hyunkyung13306** [reply]
c에 pem 키를 두어 권한 변경이 안되었던 것 같습니다.
우분트 시스템 경로로 파일을 옮긴 후 권한 수정 가능하여 접속 하였습니다 
감사합니다

**Reply by posquit0** [reply]
해결되어서 다행이네요. ㅎㅎ 👍🏼 👍🏼
